Server/src/main/java/app/item/Model.java

```diff
@@ -1,12 +1,20 @@
 package app.item;
 
+import javafx.beans.property.SimpleStringProperty;
+import weka.classifiers.Classifier;
+
 public class Model {
     private String modelName;
     private int modelID;
     private String description;
+    private SimpleStringProperty rmse;
+
+    public Model() { rmse = new SimpleStringProperty();}
+
+    public SimpleStringProperty getRMSE(){ return rmse;}
+
+    public void setRMSE(String rmse){ this.rmse.setValue(rmse);}
```

Comment:
you need to use TrainedModel to store the RMSE not Model

Reply:
I am using Model because Jeremie used Model to do his issue last sprint, I just added on top of what he did

Reply:
The RMSE is now stored in the trained model table